src/model/utils/userInfoFromToken.ts

```diff
@@ -1,4 +1,4 @@
 export class UserInfoFromToken {
     // Treat as a session class
-   constructor(public userId?: number, public companyId?: number, public companyName?: string, public username?: string, public access?: any[]) {}
+   constructor(public userId?: number, public parentCompanyId?: number, public companyId?: number, public companyName?: string, public username?: string, public access?: any[]) {}
```

Comment:
@wilzonejoe added parent companyId to the user info token, as an easy link to the company its belongs to